### PR TITLE
chore(ui-react-geo): add package boilerplate

### PR DESCRIPTION
**Pull request by sreeramsama**
#### Description of changes

Add `@aws-amplify/ui-react-geo` package boilerplate

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-geo/src/__tests__/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import * as authExports from '..';
```
Comment (ioanabrooks) [inline packages/react-geo/src/__tests__/index.spec.ts:1]: authExports -> geoExports

File: packages/react-geo/package.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    "jest": "^27.0.4",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "rimraf": "^3.0.2",
+    "rollup": "^2.70.0",
+    "rollup-plugin-node-externals": "^4.1.1",
+    "rollup-plugin-styles": "^4.0.0",
+    "rollup-plugin-terser": "^7.0.2",
+    "size-limit": "^7.0.8",
+    "ts-jest": "^27.0.3",
```
[the remark this answers is not in the document]
Comment (sreeramsama) [reply packages/react-geo/package.json]: Removed unused ts-morph import

File: packages/react-geo/package.json [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  },
+  "devDependencies": {
+    "@rollup/plugin-typescript": "^8.3.1",
+    "@size-limit/preset-big-lib": "^7.0.8",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^12.0.0",
+    "@testing-library/react-hooks": "^7.0.1",
+    "@testing-library/user-event": "^13.2.1",
+    "@types/jest": "^26.0.23",
+    "@types/qrcode": "^1.4.2",
```
[the remark this answers is not in the document]
Comment (sreeramsama) [reply packages/react-geo/package.json]: Removed unused qrcode import

File: packages/react-geo/src/__tests__/index.spec.ts (hunk 1 above)
Comment (sreeramsama) [reply packages/react-geo/src/__tests__/index.spec.ts]: oops. thanks for catching.

**Review by reesscot — APPROVED** [review]
LGTM, thanks for getting this started so quickly!

**Review by reesscot — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -7,6 +7,7 @@
     "refresh": "yarn clean && yarn setup",
     "angular": "yarn workspace @aws-amplify/ui-angular",
     "react": "yarn workspace @aws-amplify/ui-react",
+    "react-geo": "yarn workspace @aws-amplify@ui-react-geo",
```
Comment (wlee221) [inline package.json:10]:
```suggestion
    "react-geo": "yarn workspace @aws-amplify/ui-react-geo",
```

File: packages/react-geo/package.json [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    "test:ci": "yarn test && yarn check:esm",
+    "test:watch": "yarn test --watch",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@aws-amplify/ui": "5.5.8",
+    "@aws-amplify/ui-react-core": "2.1.16",
+    "classnames": "2.3.1",
+    "lodash": "4.17.21",
+    "qrcode": "1.5.0",
```
Comment (wlee221) [inline packages/react-geo/package.json:47]: Nit, and I'm pretty sure we'll get to this later, but let's remove `qrcode` so we don't forget it later.

**Review by wlee221 — APPROVED** [review]

File: packages/react-geo/package.json [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+    "test": "jest",
+    "test:ci": "yarn test && yarn check:esm",
+    "test:watch": "yarn test --watch",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@aws-amplify/ui": "5.5.8",
+    "@aws-amplify/ui-react-core": "2.1.16",
+    "classnames": "2.3.1",
+    "lodash": "4.17.21",
```
Comment (calebpollman) [inline packages/react-geo/package.json:46]: Why do we need lodash?

**Review by calebpollman — COMMENTED** [review]
Left a handful of comments that should be addressed before merge. Thanks for taking care of this!

File: packages/react-geo/package.json (hunk 6 above)
Comment (sreeramsama) [reply packages/react-geo/package.json]: Nope, removing.

File: packages/react-geo/rollup.config.js [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+import styles from 'rollup-plugin-styles';
+import externals from 'rollup-plugin-node-externals';
+
+// common config settings
+const input = ['src/index.ts'];
+const sourceMap = false;
+const tsconfig = 'tsconfig.dist.json';
+
+const config = defineConfig([
+  // CJS config
```
Comment (reesscot) [inline packages/react-geo/rollup.config.js:13]: doesn't need to be addressed here, but I wonder if we should have shared rollup configs.

**Review by reesscot — APPROVED** [review]

**Pull request merged**